zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py

```diff
@@ -32,14 +32,30 @@ def create(self, factory_ctx):
         :param factory_ctx: (dict) names available for Function instantiation
         :return: an object that implements a check function
         """
+
+        if factory_ctx.get('entity', {}).get('seeds') is not None:
```

Comment:
Also, extract `factory_ctx.get('entity', {}).get('seeds')` to a variable. It's used again below